src/Serilog.Formatting.Elasticsearch/ElasticsearchStringifyFormatter.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Serilog.Events;
+using Serilog.Formatting.Elasticsearch;
+
+namespace Host
+{
+    /// <summary>
+    /// A JSON formatter which ensures no type conflicts arieses in Elasticsearch by casting everything to strings.
+    /// 
+    /// Note that using this formatter comes at the cost that the exception tree 
+    /// with inner exceptions can grow deep.
+    /// </summary>
+    public class ElasticsearchStringifyFormatter : ExceptionAsObjectJsonFormatter
```

Comment:
The power of ES is that it can understand the different types. Although I understand what you want here, it reduces functionality in order to fix another issue. A formatter that prefixes the type I can see living in this repository, this class is a small addition for specific use cases. Can it not exist only in your project (or a separate nuget package)?

Reply:
I didn't know this was possible and would've loved if it was a part of the original package! Very cool, and solves my issue as well.

Reply:
> I didn't know this was possible and would've loved if it was a part of the original package! Very cool, and solves my issue as well.

Are you then okay with closing this PR and use it directly from your code only?